server/storage/backend/read_tx.go

```diff
@@ -120,6 +130,106 @@ func (baseReadTx *baseReadTx) UnsafeRange(bucketType Bucket, key, endKey []byte,
 	return append(k2, keys...), append(v2, vals...)
 }
 
+func bytesToRev(bytes []byte) int64 {
+	return int64(binary.BigEndian.Uint64(bytes[0:8]))
+}
+
+func (baseReadTx *baseReadTx) UnsafeRangeWithLock(bucketType Bucket, key, endKey []byte, limit int64, b Backend) ([][]byte, [][]byte) {
```

Comment:
We don't need this method, right? I think we can use bufferCopied to complete this